docs-site/docs/docs/text-cells.md

```diff
@@ -12,17 +15,39 @@ The [Text Toolbar](toolbar.md#text-cells) contains standard Rich Text Editing el
 
 It also contains a few non-Standard options.
 
-### Code and Links
+#### Code and Links
 
 The **code** button tags the currently-selected text as `code`, formatting it as monospaced. 
 
 The **link** button tags the currently-selected text as a link. Clinking on a link will reveal the URL in a tooltip, 
 allowing you to navigate to the website. 
 
-![Code and Link](images/text-cell-code-link.gif)
+![Code and Link](images/text-cell-code-link.gif){: .centered-image }
 
-### LaTeX Editor
+#### LaTeX Editor
 
 You can also input `LaTeX` into your text cells, using the button that looks like a Sigma. 
 
-![LaTeX Editor](images/text-cell-latex.gif)
\ No newline at end of file
+![LaTeX Editor](images/text-cell-latex.gif){: .centered-image }
+
+### Markdown Editing 
+The Markdown Editor allows you to edit text in traditional `atx` style markdown in a code editor and see it rendered neatly
```

Comment:
I don't think `atx` is a type of markdown – just a style of headers within markdown ("atx-style headers").

```
## This is an atx-style header

This is a non-atx style header
==============================
```